include/ipmitool/ipmi_intf.h

```diff
@@ -58,7 +58,7 @@ enum LANPLUS_SESSION_STATE {
 };
 
 
-#define IPMI_AUTHCODE_BUFFER_SIZE 20
+#define IPMI_AUTHCODE_BUFFER_SIZE 20u
```

Comment:
What exactly are you addressing with this change? Doesn't look to affect the number of warnings.

Reply:
By default, any literals are type int. This makes the literal an unsigned int, which then makes any comparisons with size_t safe (or at least not complain about signed/unsigned warnings)

Reply:
First of all this doesn't seem to actually remove any warnings. Secondly, I can't get my `gcc` to produce a warning when comparing an unsigned variable to a positive literal, can you? Lastly, if we go this way, we need to do it uniformly everywhere for all inherently unsigned literals. 
All in all, so far this looks like a superfluous change.

Reply:
Please remove this change unless you can show that it indeed helps in any way. As I said, I don't see any related warnings and can't even reproduce one 'in-vitro'.